app/routes/status.rb

```diff
@@ -0,0 +1,41 @@
+module Citygram::Routes
+  class Status < Grape::API
+    version 'v1', using: :header, vendor: 'citygram'
```

Comment:
When you issue the request to get your status, what does that request look like?  As it turns out, we can't send a header.  Perhaps we can use URL versioning for this specific case?  (It's not really a part of our API.)